addons/test_mail/tests/test_mail_mail.py

```diff
@@ -790,6 +790,22 @@ def test_mail_mail_values_email_multi(self):
             [['test.cc.1@test.example.com', 'test.cc.2@test.example.com']] * 3,
         )
 
+    def test_mail_mail_values_headers(self):
+        """ Test headers content, notably X-Odoo-Message-Id added to keep context
+        when going through stupid mail providers that change our message IDs """
```

Comment:
```suggestion
        when going through nescient mail providers that change our message IDs """
```
We can be classy and make people look up the word, which give us plausible deniability /s

Reply:
big fat ugly stupid mail providers ?

Reply:
I mean I am not against it. And if someone feels targeted, they just have to prove us wrong :shrug:

Reply:
Not like I said they are Hitler. But maybe I should ...